package.json

```diff
@@ -93,5 +93,6 @@
       "webpack": false,
       "running": false
     }
-  ]
+  ],
+  "packageManager": "npm@10.9.1+sha512.c89530d37c4baa38afd43e76a077a84b9aa63840b986426584fd5c5a54ab0a0b21bb1595c851042b733784b0b43706d36a494b4d8ae1a086a762cb8d3f95942a"
```

Comment:
Don't know if this was unintended?

Reply:
@jonrohan happy to move it to another PR, this just makes it so that (if you have corepack enabled from Node.js) that it will this version. Mine kept auto-adding a field here since I have corepack on so I figured I could just commit it lol